Data_cleaning.ipynb

```diff
@@ -0,0 +1,1915 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#importing module\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#importing the dataset by reading the csv file\n",
+    "gross = \"HighestGrossers.csv\"\n",
```

Comment:
@Muriithijoe in order for others to run this code, the csv of the data needs to be included in the same directory. Please create a folder called "Movie_Data_Cleaning_Tutorial" and upload this notebook and the CSV file in the same folder.